tests/Microsoft.Identity.Web.Test/AzureADB2COpenIDConnectEventHandlersTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Identity.Web.Test.Common.TestHelpers;
+using Microsoft.IdentityModel.Protocols.OpenIdConnect;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Identity.Web.Test
+{
+    public class AzureADB2COpenIDConnectEventHandlersTests
+    {
+        private const string PathBase = "/PathBase";
+        private const string DefaultUserFlow = "DefaultUserFlow";
```

Comment:
>DefaultUserFlow [](start = 48, length = 15)

could you use something like "b2c_1_susi", which is closer to what the value would be? makes the test easier to understand for someone who doesn't know b2c

Reply:
Agree. While the code under test doesn't specifically look for these values, updated to make it easier to understand.